Formula/czmq.rb

```diff
@@ -16,14 +16,13 @@ class Czmq < Formula
 
   head do
     url "https://github.com/zeromq/czmq.git"
```

Comment:
If there's nothing in the `head` block beside the `url` you can just move that line to one empty line beneath the `revision 3` line up top and rename the variable `head`.